web/src/test/java/com/epam/rft/atsy/cucumber/settings/positions/PositionsStepDefs.java

```diff
@@ -22,118 +24,228 @@
+    assertThat(nameValue, isEmptyOrNullString());
+  }
+
+  @Then("^the position called (.*) should appear in the list of positions$")
   public void the_new_position_appears_in_the_list_of_positions(String name) throws Throwable {
     waitForAjax();
     assertThat(getDriver().findElement(By.cssSelector("table#positions tbody")).getText(),
-        containsString(name));
+         containsString(name));
   }
 
-  @Then("^error message appears (.*)$")
+  @Then("^(.*) message should appear$")
   public void error_message_appears_message(String message) throws Throwable {
     WebDriverWait wait = new WebDriverWait(getDriver(), 15);
     wait.until(visibilityOfElementLocated(By.cssSelector("#position-form #errorMessageForCreating")));
     assertThat(getDriver().findElement(By.cssSelector("#position-form #errorMessageForCreating")).getText(),
         containsString(message));
   }
 
-  @And("^the list saved$")
-  public void the_list_saved() throws Throwable {
-    positionsListContents =
-        getDriver().findElement(By.cssSelector("table#positions tbody")).getText();
-  }
-
-  @Then("^the list of positions left unchanged$")
+  @Then("^the list of positions should left unchanged$")
   public void the_list_of_positions_left_unchanged() throws Throwable {
-
     assertThat(getDriver().findElement(By.cssSelector("table#positions tbody")).getText(),
         is(positionsListContents));
   }
 
-  @When("^the Edit button clicked on a position$")
-  public void the_Edit_button_clicked_on_a_position() throws Throwable {
-    List<WebElement> rows = getDriver().findElements(By.cssSelector("table#positions tbody tr"));
-    WebElement firstRow = rows.get(0);
-    positionName = firstRow.getText();
-    firstRow.findElement(By.cssSelector("a.edit")).click();
-  }
-
-  @Then("^the title field filled$")
+  @Then("^the title field should be filled$")
   public void the_title_field_filled() throws Throwable {
     String nameValue = getDriver().findElement(By.id("position_name")).getAttribute("value");
     assertThat(nameValue, is(positionName));
   }
 
-  @Then("^the list of positions changed$")
+  @Then("^the list of positions should be changed$")
   public void the_list_of_positions_changed() throws Throwable {
     waitForAjax();
     assertThat(getDriver().findElement(By.cssSelector("table#positions tbody")).getText(),
         is(not(containsString(positionName))));
   }
-}
+
+  @Then("^the user should get (.*) title$")
+  public void theUserGetsPageTitle(String title) throws Throwable {
+    WebElement element = driverProvider.getDriver().findElement(By.cssSelector("h1"));
+    assertThat(element.getText(), is(title));
+    assertThat(element.isDisplayed(), is(true));
+  }
+
+  @Then("^the user should get (.*) tab link$")
+  public void theUserGetsTabLink(String tab) throws Throwable {
+     WebElement element;
+     switch (tab) {
+         case "Pozíciók":
```

Comment:
There are multiple switch statements which should be replaced with a better code design.

Keep the current solution and we will see a more reusable way to achieve the same behavior.